hyperspy/datasets/artificial_data.py

```diff
@@ -0,0 +1,239 @@
+import numpy as np
+from hyperspy import components1d, components2d
+from hyperspy.signals import EELSSpectrum, Signal2D
+
+
+def get_low_loss_eel_signal():
```

Comment:
"eels" instead of "eel", to be consistent with the convention in hyperspy (EELSSpectrum, etc.)?

Reply:
Done! (Electron energy loss spectroscopy spectrum?)

Reply:
Yes, I know and it is redundant and not correct... I tried myself to use EEL spectrum and I gave up because it does look a bit weird since almost everyone used EELS spectrum. The important thing is to be consistent!